main/pipeline/pipeline.cc

```diff
@@ -200,6 +200,7 @@ ast::ParsedFile indexOne(const options::Options &opts, core::GlobalState &lgs, c
             if (opts.stopAfterPhase == options::Phase::PARSER) {
                 return emptyParsedFile(file);
             }
+
```

Comment:
```suggestion
``` 

Looks like a stray edit?

Reply:
Yep, forgot to remove that after removing the changes that I introduced earlier to track indexer errors in pipeline.cc.